esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1196,8 +1197,7 @@ void Esdl2Struct::addChildren(Esdl2Transformer *xformer, IEsdlDefObjectIterator
         {
             m_children.append(*obj);
             m_child_map.setValue(obj->queryName(), obj);
-            if (obj->queryEclName())
```

Comment:
@mayx Actually thinking about it further, although it wouldn't have been used yet, we should keep EclName support here for future compatibility.